internal/formatter/formatter.go

```diff
@@ -0,0 +1,75 @@
+package formatter
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+)
+
+type formatter func(content []byte) ([]byte, bool, error)
+
+var formatters = map[string]formatter{
+	".json": jsonFormatter,
+	".yaml": yamlFormatter,
+	".yml":  yamlFormatter,
+}
+
+// Format method formats files inside of the integration directory.
+func Format(packageRoot string, failFast bool) error {
+	err := filepath.Walk(packageRoot, func(path string, info os.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
+
+		if info.IsDir() {
+			return nil
+		}
+		err = formatFile(path, failFast)
+		if err != nil {
+			return errors.Wrapf(err, "formatting file failed (path: %s)", path)
+		}
+
+		return nil
+	})
+	if err != nil {
+		return errors.Wrap(err, "walking through the integration files failed")
+	}
+	return nil
+}
+
+func formatFile(path string, failFast bool) error {
+	file := filepath.Base(path)
+	ext := filepath.Ext(file)
+
+	content, err := ioutil.ReadFile(path)
+	if err != nil {
+		return errors.Wrap(err, "reading file content failed")
+	}
+
+	format, undefined := formatters[ext]
+	if !undefined {
```

Comment:
Shouldn't the variable name be `defined` so the `if` condition will read as, "if not defined"? Right it reads as "if not undefined", which seems inaccurate?

Reply:
Of course, it's bug. Fixed!